lib/common.sh

```diff
@@ -132,12 +132,12 @@ elif [ "${CAPM3RELEASEBRANCH}" == "release-1.1" ]; then
   CAPM3BRANCH="${CAPM3BRANCH:-release-1.1}"
   IPAMBRANCH="${IPAMBRANCH:-release-1.1}"
 else
-  CAPM3BRANCH="${CAPM3BRANCH:-main}"
-  IPAMBRANCH="${IPAMBRANCH:-main}"
+  CAPM3BRANCH="${CAPM3BRANCH:-uplift-capi-v1.2/adil}"
+  IPAMBRANCH="${IPAMBRANCH:-capi-uplift-1.2}"
 fi
 
 export IPAMPATH="${IPAMPATH:-${M3PATH}/ip-address-manager}"
```

Comment:
is this correct? because downstream ipam repo name (metal3-ipam) is different than the one from upstream ipam repo name (metal3-ip-address-manager) and since you are providing `IPAM_BASE_URL` to downstream one `IPAMPATH` should be updated accordingly